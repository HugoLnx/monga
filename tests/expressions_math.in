int main() {
	x = 10 + 15 * 20 / 5 - 30;
}
