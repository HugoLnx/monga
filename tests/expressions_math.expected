program:
  function:main,259,0
    block:
      statements:
        statement:
          attribution:
            var:x
            exp:7
              binary:1
                exp:7
                  binary:0
                    exp:0,10
                    exp:7
                      binary:3
                        exp:7
                          binary:2
                            exp:0,15
                            exp:0,20
                        exp:0,5
                exp:0,30
